cves/kernel/CVE-2015-8787.yml

```diff
@@ -356,8 +377,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The vulnerability was caused by a removal of a NULL check, which was then
+    updated in the fix
```

Comment:
Rewrite this. What does "updated in the fix" mean, how does it relate to a forgotten check?